theia-full-docker/Dockerfile

```diff
@@ -301,8 +301,7 @@ RUN chmod g+rw /home && \
     chown -R theia:theia /usr/local/go-packages
 
 # Theia application
-## Needed for node-gyp, nsfw build
-RUN apt-get clean && \
+RUN apt-get clean libsecret-1-dev && \
```

Comment:
@marcdumais-work we clean `libsecret-1-dev` but do we need to include it with the_`common deps_:
- https://github.com/theia-ide/theia-apps/pull/482/files#diff-cd78a70fc9e82ba3771eedd3c11041a1424f2471ca55b4e4d084ed8ea6b72d43R11

Reply:
done

Reply:
> and the CI is passing

Note: our conservative build system works against us here. Images that I have not changes are not rebuilt, which hides the fact they need update. E.g. the Python image. The nightly build will show the truth :)